NEWS.md

```diff
@@ -211,6 +215,10 @@ up correct aspect ratio, and draws a graticule.
 
 * Legends no longer try and use set aesthetics that are not length one (#1932).
 
+* All scales that are not position scales now have an `aesthetics` argument
+  that can be used to set the aesthetics the scale works with, regardles of
+  the name of the scale function. (@clauswilke)
```

Comment:
For this version, maybe we could just add that argument to colour scales? You'll still be able to set it via `...` but it won't be so in your face.

Reply:
Ugh, of course, blech.  What we just leave off the non-colour aesthetics for now and consider the user API in the future? I'd like to get this into this release, but I don't have the time to think through any big API changes.

Reply:
In that case, how do you feel about exporting `manual_scale()` (possibly under a different name, e.g. `scale_discrete_manual()`)?

https://github.com/tidyverse/ggplot2/blob/0dab0442b45dbcb298af02db0d4b06cafd964897/R/scale-manual.r#L74

The problem is that there is currently literally no simple way of defining, for example, a discrete shape scale for an aesthetic other than `shape`. One always has to reimplement `manual_scale()`, and it's the reason I invented `scale_discrete_manual()` in ggridges:
https://github.com/clauswilke/ggridges/blob/4e94e9a32b976c5a2eedc6fa04b104178f3e9767/R/scale-.R#L26

Reply:
Yeah, that sounds good, and I think that name is fine.

Reply:
Ok, to summarize, I will:

1. Revert addition of `aesthetics` argument for non-color scales.
2. Add and export `scale_discrete_manual()` which works just like `manual_scale()`.

I'll try to get this done by Friday.